### PR TITLE
feat: add Content to Info block

### PR DESCRIPTION
**Pull request by aeksandla**

### CODE REVIEW

**Review by yuberdysheva — CHANGES_REQUESTED** [review]

File: src/blocks/Info/Info.tsx [hunk 1]
```diff
@@ -14,53 +13,58 @@ const b = block('info-block');
...
+                                title={sectionsTitle || rightContent?.title}
+                                text={rightContent?.text}
+                                links={
+                                    links?.map((link) => ({
+                                        ...link,
+                                        arrow: true,
+                                        theme: 'normal',
+                                    })) || rightContent?.links
+                                }
+                                theme={contentTheme}
```
Comment (gorgeousvlad) [inline src/blocks/Info/Info.tsx:61]: please put all the same props form this <Content> blocks to one variable

File: src/blocks/Info/Info.tsx [hunk 2]
```diff
@@ -14,53 +13,58 @@ const b = block('info-block');
...
         title,
         sectionsTitle,
         links,
+        rightContent = {},
+        leftContent = {},
     } = props;
 
     const {themeValue: theme} = useContext(ThemeValueContext);
+    const contentTheme = blockTheme === 'dark' ? 'dark' : 'default';
+    const sizes = {md: 6, all: 12};
```
Comment (gorgeousvlad) [inline src/blocks/Info/Info.tsx:27]: can we declare this constant outside of the component function?

File: src/blocks/Info/README.md [hunk 3]
```diff
@@ -2,14 +2,18 @@ Info block
 
 `type: 'info-block'`
 
-`title: string` — Title on the right
+`title: string` — Title on the right. DEPRECATED
 
-`sectionsTitle: string` — Title on the left
+`sectionsTitle: string` — Title on the left. DEPRECATED
```
[the remark this answers is not in the document]
Comment (gorgeousvlad) [reply src/blocks/Info/README.md:7]: lets just accumulate enough changes before major version batching

File: src/blocks/Info/README.md [hunk 4]
```diff
@@ -2,14 +2,18 @@ Info block
...
 
 `backgroundColor?: string` — Fill color
 
-`buttons?: Button[]` — Buttons on the left
+`buttons?: Button[]` — Buttons on the left. DEPRECATED
```
Comment (gorgeousvlad) [inline src/blocks/Info/README.md:11]: please mark this props with `/** @deprecated **/` in code

**Review by gorgeousvlad — CHANGES_REQUESTED** [review]

File: src/blocks/Info/__stories__/data.json [hunk 5]
```diff
@@ -1,35 +1,153 @@
...
-      "sectionsTitle": "Useful links",
-      "links": [
-        {
-          "text": "Find a partner",
-          "url": "/partners/find"
-        },
-        {
-          "text": "Billing",
-          "url": "/prices"
+        "text": "<p><strong>Ut enim ad minim veniam</strong> <a href=\"https://example.com\">quis nostrud</a> exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.</p> ",
```
[the remark this answers is not in the document]
Comment (aeksandla) [reply src/blocks/Info/__stories__/data.json:12]: yfm can be used here

File: src/blocks/Info/Info.scss [hunk 6]
```diff
@@ -4,58 +4,41 @@
...
-    &__content-title,
-    &__sections-title {
-        @include heading2();
-    }
-
-    &__content {
+    &__container {
         background-color: var(--yc-color-base-special);
-        padding: $indentXXL $indentXL $indentXXXL;
+        padding: $indentXXL $indentXL 112px;
```
[the remark this answers is not in the document]
Comment (aeksandla) [reply src/blocks/Info/Info.scss]: fixed

File: src/blocks/Info/README.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (aeksandla) [reply src/blocks/Info/README.md]: remove this fields from the README

File: src/blocks/Info/__stories__/data.json [hunk 7]
```diff
@@ -1,35 +1,153 @@
...
+            "text": "Button",
+            "theme": "action",
+            "url": "#"
+          }
+        ]
+      },
+      "leftContent": {
+        "title": "Lorem ipsum dolor sit amet, consectetur adipiscing elit",
+        "text": "<p><strong>Ut enim ad minim veniam</strong> <a href=\"https://example.com\">quis nostrud</a> exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.</p> ",
+        "additionalInfo": "<p> Duis aute irure dolor in <a href=\"https://example.com\">reprehenderit</a>n voluptate velit esse cillum dolore eu fugiat nulla pariatur.</p> ",
```
[the remark this answers is not in the document]
Comment (aeksandla) [reply src/blocks/Info/__stories__/data.json]: create common object with same texts

File: src/models/constructor-items/blocks.ts [hunk 8]
```diff
@@ -238,10 +238,12 @@ export interface PreviewBlockProps extends Animatable {
 export interface InfoBlockProps {
     theme?: TextTheme;
     backgroundColor?: ThemeSupporting<string>;
-    title: string;
+    title?: string;
```
[the remark this answers is not in the document]
Comment (aeksandla) [reply src/models/constructor-items/blocks.ts:242]: add

File: src/blocks/Info/Info.scss [hunk 9]
```diff
@@ -4,58 +4,41 @@
...
     }
 
-    &__button {
-        margin-top: $indentXS;
-
-        &:not(:last-child) {
-            margin-right: $indentXS;
-        }
+    &__content &__title a {
+        text-decoration: none;
```
[the remark this answers is not in the document]
Comment (aeksandla) [reply src/blocks/Info/Info.scss]: moved to Content

File: src/blocks/Info/Info.tsx (hunk 1 above)
Comment (aeksandla) [reply src/blocks/Info/Info.tsx:62]: fixed

File: src/blocks/Info/README.md (hunk 4 above)
Comment (aeksandla) [reply src/blocks/Info/README.md]: add

File: src/blocks/Info/Info.tsx (hunk 1 above)
Comment (gorgeousvlad) [reply src/blocks/Info/Info.tsx:62]: i mean same props of these two `<Content>` components: theme, additionalInfo, colSizes, etc.

Comment (aeksandla) [reply src/blocks/Info/Info.tsx]: add commonProps with theme, className, colSizes. additionalInfo is different for two `<Content>`

File: src/blocks/Info/Info.scss (hunk 6 above)
[the remark this answers is not in the document]
Comment (aeksandla) [reply src/blocks/Info/Info.scss]: fixed

**Review by gorgeousvlad — APPROVED** [review]

File: src/blocks/Info/__stories__/data.json (hunk 5 above)
[the remark this answers is not in the document]
Comment (aeksandla) [reply src/blocks/Info/__stories__/data.json]: fixed

**Review by yuberdysheva — APPROVED** [review]

**Pull request merged**